dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3602,18 +3614,25 @@ def test_correct_null_row_counts(self):
             }
         )
         profile = dp.StructuredProfiler(data, options=profiler_options)
-        self.assertEqual(2, profile.row_has_null_count)
-        self.assertEqual(0.25, profile._get_row_has_null_ratio())
-        self.assertEqual(2, profile.row_is_null_count)
-        self.assertEqual(0.25, profile._get_row_is_null_ratio())
+        self.assertEqual(10, profile.row_has_null_count)
+        self.assertEqual(0.5, profile._get_row_has_null_ratio())
+        self.assertEqual(0, profile.row_is_null_count)
+        self.assertEqual(0, profile._get_row_is_null_ratio())
 
-        file_path = os.path.join(test_root_path, "data", "csv/iris-with-null-rows.csv")
-        data = pd.read_csv(file_path)
-        profile = dp.StructuredProfiler(data, options=profiler_options)
-        self.assertEqual(13, profile.row_has_null_count)
-        self.assertEqual(13 / 24, profile._get_row_has_null_ratio())
-        self.assertEqual(3, profile.row_is_null_count)
-        self.assertEqual(3 / 24, profile._get_row_is_null_ratio())
+        # I commented out these lines of code, because they are a second
+        # test of the functions tested in the last four lines of code.
+        # Since we intend to use only the setUpClass data, there is no
+        # reason to keep these in, or test those functions a second time.
+
+        # file_path = os.path.join(test_root_path, "data", "csv/iris-with-null-rows.csv")
+        # data = pd.read_csv(file_path)
+        # data = self.data
+
+        # profile = dp.StructuredProfiler(data, options=profiler_options)
+        # self.assertEqual(13, profile.row_has_null_count)
+        # self.assertEqual(13 / 24, profile._get_row_has_null_ratio())
+        # self.assertEqual(3, profile.row_is_null_count)
+        # self.assertEqual(3 / 24, profile._get_row_is_null_ratio())
```

Comment:
i'd recommend either deleting or keeping -- not just keeping commented out code in the file